ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -56,7 +56,13 @@ kubevirt) export KUBEVIRT_KUBECONFIG=${HOME}/.kube/config;;
 vsphere) ;;
 openstack-osuosl) ;;
 openstack-ppc64le) ;;
-openstack*) export OS_CLIENT_CONFIG_FILE=${SHARED_DIR}/clouds.yaml ;;
+openstack*)
+  export OS_CLIENT_CONFIG_FILE=${SHARED_DIR}/clouds.yaml
+  if [ -f "${SHARED_DIR}/osp-ca.crt" ]; then
+    # Using sed as the installer is not consuming OS_CACERT env var
+    sed -i "s+cacert: .*+cacert: ${SHARED_DIR}/osp-ca.crt+" "${SHARED_DIR}/clouds.yaml"
```

Comment:
cacert is now handled by https://github.com/openshift/release/pull/19392; you should be able to remove this code